src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -212,67 +212,50 @@ public void invoke() {
   }
 
   public void getEntityBundleAndLoadPageTop() {
-    synapseClient.getDatasetScriptElementContent(
```

Comment:
This is a terrible diff.  In reality, I just removed the getDatasetScriptElementContent block (looking at the raw source might be preferable).